### ISSUE TITLE
[java] CommentDefaultAccessModifier ignoredAnnotations should include "org.junit.jupiter.api.extension.RegisterExtension" by default

### ISSUE THREAD

**Issue by koalalam** [pmd/pmd#4273]
**Affects PMD Version:** 

6.52.0

**Rule:**

CommentDefaultAccessModifier

https://pmd.sourceforge.io/pmd-6.52.0/pmd_rules_java_codestyle.html#commentdefaultaccessmodifier

**Description:**

**Code Sample demonstrating the issue:**

```java

import com.github.tomakehurst.wiremock.common.ConsoleNotifier;
import com.github.tomakehurst.wiremock.junit5.WireMockExtension;
import org.junit.jupiter.api.extension.RegisterExtension;

class SomeTest {

  @RegisterExtension
  static final WireMockExtension WIRE_MOCK_EXTENSION =
      WireMockExtension.newInstance()
          .options(wireMockConfig().dynamicPort().notifier(new ConsoleNotifier(true)))
          .build();
}
```

**Expected outcome:**

Default value of "ignoredAnnotations" should include "org.junit.jupiter.api.extension.RegisterExtension"

**Comment by nirvikpatel** [comment]
Can i pick up this issue @koalalam ?

**Comment by adangel** [comment]
See also #3859
